pkg/project/project.go

```diff
@@ -5,7 +5,7 @@ var (
 	gitSHA             = "n/a"
 	name        string = "cert-operator"
 	source      string = "https://github.com/giantswarm/cert-operator"
-	version            = "1.0.1-dev"
+	version            = "0.1.0"
```

Comment:
```suggestion
	version            = "0.2.0"
```

Reply:
This won't work, this version is actually use in the reconciliation loop ([see](https://github.com/giantswarm/cert-operator/blob/master/service/controller/resource_set.go#L148)) to match against CRs and they all have `0.1.0`.

Reply:
Oh, right. So you want to align it back with the tag. Gotcha.